Modify default set of favorite apps to match what we ship in Solaris

Not suitable for upstream

--- a/data/org.gnome.shell.gschema.xml.in.in
+++ b/data/org.gnome.shell.gschema.xml.in.in
@@ -31,7 +31,7 @@
       </_description>
     </key>
     <key name="favorite-apps" type="as">
-      <default>[ 'epiphany.desktop', 'evolution.desktop', 'rhythmbox.desktop', 'shotwell.desktop', 'org.gnome.Nautilus.desktop', 'org.gnome.Software.desktop' ]</default>
+      <default>[ 'firefox.desktop', 'thunderbird.desktop', 'pidgin.desktop', 'gnome-terminal.desktop', 'org.gnome.Nautilus.desktop', 'os-next-steps.desktop' ]</default>
       <_summary>List of desktop file IDs for favorite applications</_summary>
       <_description>
         The applications corresponding to these identifiers
--- a/js/ui/appFavorites.js
+++ b/js/ui/appFavorites.js
@@ -31,6 +31,7 @@ const RENAMED_DESKTOP_IDS = {
     'gnotravex.desktop': 'gnome-tetravex.desktop',
     'gnotski.desktop': 'gnome-klotski.desktop',
     'gtali.desktop': 'tali.desktop',
+    'mozilla-firefox.desktop': 'firefox.desktop',
     'nautilus.desktop': 'org.gnome.Nautilus.desktop',
     'polari.desktop': 'org.gnome.Polari.desktop',
     'totem.desktop': 'org.gnome.Totem.desktop',
